data_steward/validation/metrics/required_labs.py

```diff
@@ -54,6 +75,35 @@ def load_measurement_concept_sets_descendants_table(project_id, dataset_id):
     :return: None
     """
 
+    descendants_table_name = f'{project_id}.{dataset_id}.{MEASUREMENT_CONCEPT_SETS_DESCENDANTS_TABLE}'
+    vocab_table_name = f'{project_id}.{dataset_id}.{common.VOCABULARY}'
```

Comment:
The `concept` table and the `concept_ancestor` table need to be verified to exist in `dataset_id`. If they don't, they need to be copied from the dataset `common.VOCABULARY` into `dataset_id`.

Reply:
Okay, I was off on that one, I understand now and will make the changes.